oggm/utils/_downloads.py

```diff
@@ -45,6 +45,7 @@
                              DownloadVerificationFailedException,
                              DownloadCredentialsMissingException,
                              HttpDownloadError, HttpContentTooShortError)
+from oggm import utils
```

Comment:
There is a risk of circular dependencies here - utils is in fact three modules including this one. Remove this line and import directly in the function itself with `from oggm.utils._workflow import robust_tar_extract`

Reply:
(reason: not so good design of a huge utils module...)